r4/fhirpath/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
+    "image": "ballerina/ballerina-devcontainer:2201.5.0",
```

Comment:
We are migrating Bal version to 6.0.0, could you please check the Choreo support version and update here

Reply:
Fixed in 28ab50418e064931acc8a5e17b22cbfa773c827a